database/oneOff/merge-series-table.sql

```diff
@@ -0,0 +1,31 @@
+DELIMITER //
+
+create procedure handleSeries()
+begin
+    declare mofferingid int default 0;
+    declare seriesIdvalue int default 0;
+    declare done int default 0;
+    declare cur1 cursor for select distinct multipleOfferingId from event where multipleOfferingId is not null;
+    declare continue handler for sqlstate '02000' set done = 1;
+    update event set seriesId=recurringId where recurringId is not null;
```

Comment:
This line can be removed and the one on line 13 moved up here

Reply:
Solved.